templates/facetDemoScript.template.hbs

```diff
@@ -0,0 +1,165 @@
+import {
+  createPublicClient,
+  createWalletClient,
+  getContract,
+  http,
+  parseUnits,
+  Narrow,
+  zeroAddress
+} from 'viem'
+import { privateKeyToAccount } from 'viem/accounts'
+import { randomBytes } from 'crypto'
+import dotenv from 'dotenv'
+import deployments from '../../deployments/mainnet.staging.json'
+import erc20Artifact from '../../out/ERC20/ERC20.sol/ERC20.json'
+import {{camelCase name}}FacetArtifact from '../../out/{{titleCase name}}Facet.sol/{{titleCase name}}Facet.json'
+import config from '../../config/{{camelCase name}}.json'
+import { {{titleCase name}}Facet, ILiFi } from '../../typechain'
+import {
+  getEnvVar,
+  normalizePrivateKey
+} from './utils/demoScriptHelpers'
+import { getRpcUrl, getViemChain, SupportedChain } from './utils/demoScriptChainConfig'
+
+// #region ABIs
+
+const ERC20_ABI = erc20Artifact.abi as Narrow<typeof erc20Artifact.abi>
+const {{constantCase name}}_FACET_ABI = {{camelCase name}}FacetArtifact.abi as Narrow<typeof {{camelCase name}}FacetArtifact.abi>
+const EXAMPLE_ABI = [{...}] // optional define example ABI
+
+// #endregion
+
+dotenv.config()
+
+async function main() {
+  // === Set up parameters, retrieve env vars and set up clients ===
+  // `srcChain` is the source chain for the operation (change as needed).
+  // RPC URL and private key are dynamically fetched based on the chain.
+  const srcChain: SupportedChain = "mainnet"; // Set source chain
+  const RPC_URL = getRpcUrl(srcChain)
+  const PRIVATE_KEY = getEnvVar('PRIVATE_KEY')
+  const typedPrivateKey = normalizePrivateKey(PRIVATE_KEY)
+
+  const publicClient = createPublicClient({
+    chain: getViemChain(srcChain),
+    transport: http(RPC_URL),
+  })
+  const walletAccount = privateKeyToAccount(typedPrivateKey)
+  const walletClient = createWalletClient({
+    chain: getViemChain(srcChain),
+    transport: http(RPC_URL),
+    account: walletAccount,
+  })
+  const signerAddress = walletAccount.address
+
+  // === Contract addresses ===
+  // LIFI_ADDRESS is the address of the LiFi Diamond contract for bridging.
+  // SRC_TOKEN_ADDRESS is the token to be bridged (set its address for the source chain).
+  // EXAMPLE_ADDRESS is a custom contract address for a contract that your facet interacts with.
```

Comment:
Is this the "bridge router"? Example address is not a good name imho as it doesnt specify anything and doesnt give any guidance on what kind of address may be required here. Try to be more specific and also if it's optional, consider making it a comment by default